components/GoalItem.js

```diff
@@ -6,6 +6,7 @@
     <View style={styles.goalItem}>
       <Pressable android_ripple={{color:'#210466'}}
       onPress={props.onDeleteItem.bind(this,props.id)}
+
       style={(pressData)=>pressData.pressed && styles.pressedItemi}
```

Comment:
wrong style name styles.pressedItem 
```suggestion
      style={(pressData)=>pressData.pressed && styles.pressedItem}
```